configs/mainnet.yaml

```diff
@@ -195,9 +195,3 @@ MAX_BYTES_PER_INCLUSION_LIST: 8192
 # ---------------------------------------------------------------
 
 BLOB_SCHEDULE:
```

Comment:
sorry for late comment, but shouldn't this rather be

```yaml
  BLOB_SCHEDULE: []
```

Reply:
Err yes, you're right. Will fix this, thanks!